### DISCUSSION TITLE
VLESS + TCP + TLS的安全性

### DISCUSSION THREAD

**Discussion by secXsQuared** [v2fly/v2ray-core#619] [General]
请问现在用VLESS + TCP + TLS是不是安全性和之前的Vmess + WS + TLS一样，但是速度快因为不需要websocket？ 就是用 VLESS的fallback功能+一个本地的webserver。

**Comment by pengxiaoyan8888** [comment]
同问，我也想知道vless+tcp+tls是否与xtls一样容易被墙
